### PR TITLE
No more asset pipelining

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @LukasReschke

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Another ref https://github.com/nextcloud/documentation/blob/968eb3cb3fd5c81d852dcf7357406cc2dc2b07ce/admin_manual/configuration_server/oc_server_tuning.rst#enable-javascript-and-css-asset-management

**Comment by nickvergessen** [comment]
Fixed, thanks for checking

File: admin_manual/configuration_server/index.rst [hunk 1]
```diff
@@ -20,9 +20,8 @@ Nextcloud Server Configuration
    harden_server
    reverse_proxy_configuration 
    thirdparty_php_configuration
-   js_css_asset_management_configuration
    automatic_configuration
-   oc_server_tuning
```
Comment (ChristophWurst) [inline admin_manual/configuration_server/index.rst]: https://github.com/nextcloud/documentation/blob/a50bad751a382d9c7bcd16026793e6d40e332211/go.php#L18 :see_no_evil:

**Comment by ChristophWurst** [comment]
What do we have CI for if it doesn't fail? hehe

**Comment by nickvergessen** [comment]
Well, I guess there is no test for the go.php

**Comment by ChristophWurst** [comment]
:+1: looks good now

**Comment by ChristophWurst** [comment]
> You're not authorized to merge this pull request.

:cry:

**Pull request merged**